.gitlab-ci.yml

```diff
@@ -200,6 +200,15 @@ deploy-rococo-ink-docs:
     DOMAIN: parity-testnet
     APP: substrate-faucet-rococo-ink-docs
 
+deploy-westend-web:
+  stage: deploy
+  <<: *deploy-prod-refs
+  extends: .deploy-with-argocd
+  environment: westend-web
+  variables:
+    DOMAIN: parity-testnet
+    APP: substrate-faucet-westend-web
+
```

Comment:
let's drop this section, as we won't be deploying westend web faucet separately from its matrix counterpart